### PR TITLE
Feat(Task-3): As a developer, I want to manage the screen change easily

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (eloydotco) [inline .vscode/settings.json:1]: add .vscode folder to .gitignore

File: src/app/app.module.js [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
 import ngAria from 'angular-aria';
 
-angular.module('app', [ngMaterial, ngAnimate, ngAria]);
+angular.module('app', [ngMaterial, ngAnimate, ngAria]).component('app', appComponent);
+const myApp = angular.module('app', [uirouter, ngMaterial, ngAnimate, ngAria])
+
+myApp.component('hello', helloComp);
+myApp.component('world', worldComp);
+
+myApp.config(['$stateProvider', function(provider) {
```
Comment (gbtec-ext-jorgevillar) [inline src/app/app.module.js]: Create the config in another file an import it here

File: src/app/app.module.js [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+import uirouter from '@uirouter/angularjs';
+import helloComp from './hello/hello';
+import worldComp from './world/world';
 import ngMaterial from 'angular-material';
 import ngAnimate from 'angular-animate';
 import ngAria from 'angular-aria';
 
-angular.module('app', [ngMaterial, ngAnimate, ngAria]);
+angular.module('app', [ngMaterial, ngAnimate, ngAria]).component('app', appComponent);
+const myApp = angular.module('app', [uirouter, ngMaterial, ngAnimate, ngAria])
```
Comment (gbtec-ext-jorgevillar) [inline src/app/app.module.js]: Create the modules in this way
image

File: src/app/hello/hello.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import template from './hello.html';
+import './hello.css';
+const AppComponent = {
+    template
+};
+export default AppComponent;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/hello/hello.js]: And components like this

image

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]

**Review by eloydotco — APPROVED** [review]

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**